scripts/reindex.py

```diff
@@ -120,16 +120,15 @@ def main(argv: List[str]):
             matches = fnmatch.filter(sources, source_glob)
             if matches:
                 globs_matched.add(source_glob)
-            logger.debug('Matched sources %r in catalog %r', matches, catalog)
+            logger.debug('Source glob %r matched sources %r in catalog %r',
+                         source_glob, matches, catalog)
```

Comment:
While testing, I observed this log message to be useless without explaining which source was being matched.